### DISCUSSION TITLE
Start a new discussion

### DISCUSSION THREAD

**Discussion by macoptions** [macoptions/Get_Badge_repo#2] [Q&A] [answered]
[Pull requests](https://github.com/macoptions/Get_Badge_repo/pulls)
[Discussions](https://github.com/macoptions/Get_Badge_repo/discussions)
[Actions](https://github.com/macoptions/Get_Badge_repo/actions)
[Projects](https://github.com/macoptions/Get_Badge_repo/projects)
[Wiki](https://github.com/macoptions/Get_Badge_repo/wiki)
[Security](https://github.com/macoptions/Get_Badge_repo/security)
[Insights](https://github.com/macoptions/Get_Badge_repo/pulse)
[Settings](https://github.com/macoptions/Get_Badge_repo/settings)
Start a new discussion
🙏
Q&A
Ask the community for help
Answers enabled
If this doesn’t look right you can [choose a different category.](https://github.com/macoptions/Get_Badge_repo/discussions/new/choose)

**Comment by namdodayne** [comment, marked as answer]
GET badge
